extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -648,7 +678,7 @@ export class ConsentUI {
   resetAnimationStyles_() {
     setStyles(this.parent_, {
       transform: '',
-      transition: '',
+      '--lightbox-height': '',
```

Comment:
I don't recall why we need to reset `transition`, but comment says it's added by the fixed layer. Can we remove it safely?

Reply:
Ahh, my mistake, didn't look into it thoroughly enough.